std_msgs/Header header

geometry_msgs/Vector3 gain_p
geometry_msgs/Vector3 gain_i
geometry_msgs/Vector3 gain_d

geometry_msgs/Vector3 error
geometry_msgs/Vector3 error_integral
geometry_msgs/Vector3 error_derivative